src/configs/git_commit.rs

```diff
@@ -0,0 +1,28 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct GitCommitConfig<'a> {
+    pub hash: SegmentConfig<'a>,
+    pub prefix: &'a str,
+    pub suffix: &'a str,
+    pub style: Style,
+    pub disabled: bool,
+    pub commit_hash_length: usize,
```

Comment:
Nitpick: could we have configuration values unique to the module moved to the top of the list?